Source/CarthageKit/Xcode.swift

```diff
@@ -228,6 +228,12 @@ public func xcodebuildTask(task: String, _ buildArguments: BuildArguments) -> Ta
 	return Task("/usr/bin/xcrun", arguments: buildArguments.arguments + [ task ])
 }
 
+/// Creates a task description for executing `xcodebuild` with the given
+/// arguments.
+public func xcodebuildTask(task: String, _ task2: String, _ buildArguments: BuildArguments) -> Task {
+	return Task("/usr/bin/xcrun", arguments: buildArguments.arguments + [ task, task2 ])
```

Comment:
I overloaded it because it's public API and I wasn't sure if there were any issues with changing public API like that. But I do prefer the array of tasks option, with an overload to take a single task for convenience.

I actually wish we could say `func xcodebuildTask(tasks: String..., _ buildArguments: BuildArguments) -> Task` but you can't use variadic args before an unnamed parameter :(